node/src/bin/manager.rs

```diff
@@ -582,6 +605,30 @@ async fn main() {
                 sleep,
             )
         }
+        TestRun {
```

Comment:
Any objection to calling this Run and having stop block as an optional flag?

Reply:
I'm with @mangas, `graphman run` feels better.

About your concerns @sduchesneau , `graphman` is a tool that is deemed for "management of graph-node" instances, so I don't think it will be confused as production means to run a graph node.

@lutter What do you think about this, do you think there could be some confusions if we use `graphman run`?

Reply:
I'm +1 on `run`